std-bits/table/src/main/resources/META-INF/native-image/org/enso/table/resource-config.json

```diff
@@ -0,0 +1,96 @@
+{
+  "resources":{
+
+  "includes":[
+    {
+      "pattern": "org/apache/poi/schemas/ooxml/system/ooxml/.*\\.xsb$"
+    },
+    {
+      "pattern": "org/apache/poi/schemas/ooxml/element/.*\\.xsb$"
+    },
+    {
+      "pattern": "org/apache/xmlbeans/metadata/.*\\.xsb$"
+    },
+    {
+      "pattern": "org/apache/xmlbeans/metadata/.*\\.xsd$"
+    },
+    {
+    "pattern":"\\QMETA-INF/log4j-provider.properties\\E"
+  }, {
+    "pattern":"\\QMETA-INF/maven/org.xerial/sqlite-jdbc/pom.properties\\E"
+  }, {
+    "pattern":"\\QMETA-INF/org/enso/interpreter/node/expression/builtin/BuiltinMethods.metadata\\E"
+  }, {
+    "pattern":"\\QMETA-INF/org/enso/interpreter/node/expression/builtin/BuiltinTypes.metadata\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/ch.qos.logback.classic.spi.Configurator\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.lang.Cloneable\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.lang.System$LoggerFinder\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.net.ContentHandlerFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.nio.charset.spi.CharsetProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.sql.Driver\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.time.zone.ZoneRulesProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/java.util.spi.ResourceBundleControlProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/javax.xml.parsers.DocumentBuilderFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/javax.xml.parsers.SAXParserFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/javax.xml.transform.TransformerFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/javax.xml.xpath.XPathFactory\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.apache.logging.log4j.spi.Provider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.apache.logging.log4j.util.PropertySource\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.apache.tika.detect.EncodingDetector\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.enso.base.file_format.FileFormatSPI\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.enso.base.read.ReadManyReturnSPI\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.enso.persist.Persistance\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.slf4j.spi.SLF4JServiceProvider\\E"
+  }, {
+    "pattern":"\\QMETA-INF/services/org.w3c.dom.DOMImplementationSourceList\\E"
+  }, {
+    "pattern":"\\Qlog4j2.StatusLogger.properties\\E"
+  }, {
+    "pattern":"\\Qlog4j2.component.properties\\E"
+  }, {
+    "pattern":"\\Qlog4j2.system.properties\\E"
+  }, {
+    "pattern":"\\Qlogback.xml\\E"
+  }, {
+    "pattern":"\\Qreference.conf\\E"
+  }, {
+    "pattern":"\\Qsqlite-jdbc.properties\\E"
+  }, {
+    "pattern":"\\Qurlhandler.props\\E"
+  }]},
+  "bundles":[{
+    "name":"com.sun.org.apache.xalan.internal.xsltc.compiler.util.ErrorMessages",
```

Comment:
Xalan! Isn't it generating bytecode on the fly? I remember these used to be [Gregor Samsa class](https://github.com/apache/xalan-java/blob/6fd607c78ebb07ded14521a8ab460a7f4ba4fc1b/xalan/src/main/java/org/apache/xalan/xsltc/compiler/XSLTC.java#L337) which resembled not only transformation but also bytecode generator. How that's gonna work with NI?...